docs: add docs for editing permissions (#97)
GameFabric/docs: src/multiplayer-servers/getting-started/editing-permissions.md

# Editing Permissions

## Overview

> In GameFabric, you can manage permissions for users, groups, and roles. This allows you to control who can access features and resources within your GameFabric environment. 

Reviewer: how about:

> GameFabric uses RBAC (Role-Based Access Control) to manage who can access specific features and resources within your environment. This is achieved by defining permissions for particular roles, which are then assigned to groups of users.
?

## Groups
A group is a collection of users that can be assigned roles. 
Existing groups are listed on the **Groups/Users** page of the GameFabric UI, under the **Groups** tab. 
From here, you can view all groups, create new ones, or edit existing groups.
![groups-overview.png](images/permissions/groups-overview.png)

### Add Roles To A Group
To add roles to a group, click the **Edit** button next to the desired group. 
This will open the group edit modal, where you can add or remove roles. Select the roles you want to assign, then click **Save**.
![add-role-to-group.png](images/permissions/add-role-to-group.png)

### Add User To A Group
In the group edit modal, click the **Users** tab. Here, you can search for users by their email address. 
Click **Add User**, select the desired user, and then click **Save** to apply your changes.  
![add-user-to-group.png](images/permissions/add-user-to-group.png)

#### Remove User From Group
To remove a user, click the **X** next to their email address. Then click **Save** to confirm.  
![remove-user-from-group.png](images/permissions/remove-user-from-group.png)

## User
A user is an individual account that can be assigned to groups and roles. 

> Users are managed on the **Groups/Users** page of the GameFabric UI, under the **Users** tab. 

Author: To change the order we would first have to change it in the UI, so right now I would leave it as is. 
As for the SSO, I added an info box. Wdyt?

To edit a user, click the **Edit** button next to their name. In the user edit modal, select the groups to which the user should belong. 
Click **Save** to apply your changes.
![add-groups-to-user.png](images/permissions/add-groups-to-user.png)

## Roles

> A role is a collection of permissions that can be assigned to users or groups. 

Author: True, I adjusted the text

# Editing Permissions

> ## Overview
> GameFabric uses RBAC (Role-Based Access Control) to manage who can access specific features and resources within your environment. This is achieved by defining permissions for particular roles, which are then assigned to groups of users.

Reviewer 2: ```suggestion
## Overview
GameFabric uses RBAC (Role-Based Access Control) to manage who can access specific features and resources within your GameFabric installation. This is achieved by defining permissions for particular roles, which are then assigned to groups of users.
```

Environment isn't wrong but might be confusing because of our environments we have

> ## Overview
> GameFabric uses RBAC (Role-Based Access Control) to manage who can access specific features and resources within your environment. This is achieved by defining permissions for particular roles, which are then assigned to groups of users.

Reviewer 2: ```suggestion
## Overview
GameFabric uses RBAC (Role-Based Access Control) to manage who can access specific features and resources within your environment. This is achieved by defining a role—a set of permissions—that is then assigned to user groups.
```

You technically don't define permissions, but maybe i'm to nitty gritty here :D Thoughts from others?

To edit permissions, navigate to the **Access Management** section in the GameFabric UI and select either the **Groups/Users** or **Roles** page.

## Group

> A group is a collection of users that can be assigned roles.

Reviewer 2: ```suggestion
A group is a collection of users that can be assigned to roles.
```

Existing groups are listed on the **Groups/Users** page of the GameFabric UI, under the **Groups** tab.
From here, you can view all groups, create new ones, or edit existing groups.
![groups-overview.png](images/permissions/groups-overview.png)

> ### Add Roles To A Group

Reviewer: yes please

#### Remove User From Group
To remove a user, click the **X** next to their email address. Then click **Save** to confirm.
![remove-user-from-group.png](images/permissions/remove-user-from-group.png)

## User
A user is an individual account that can be assigned to groups.
Users are managed on the **Groups/Users** page of the GameFabric UI, under the **Users** tab.
To edit a user, click the **Edit** button next to their name. In the user edit modal, select the groups to which the user should belong.
Click **Save** to apply your changes.
![add-groups-to-user.png](images/permissions/add-groups-to-user.png)

::: info
To set up a Third-Party OIDC Provider, refer to the [Setup Third-Party OAuth Integration](setup-third-party-oauth.md) section.
:::

## Role
A role is a collection of permissions that can be assigned to groups.
To edit roles, navigate to the **Access Management** section in the GameFabric UI and select the **Roles** page.
To modify an existing role, click the **Edit** button next to it.
![roles-overview.png](images/permissions/roles-overview.png)

Each role has a set of permissions that define the actions available to users or groups assigned to that role.

> The resources listed in the table correspond to the REST API. To grant a role access to a resource, check the box in the respective column.

Author: ```suggestion
To grant a role permission to a resource, check the box in the respective column.
```
Does that do the trick for you?
